docs/modules/general/pages/implementation-details/bootstrapping.adoc

```diff
@@ -0,0 +1,584 @@
+= OneCX Microfrontend Content Bootstrapping
+
+== Overview
+// TODO: Add link to MFE loading
```

Comment:
What do you think of a general introduction of Microfrontend Bootrapping?

Bootstrapping a microfrontend involves setting up the initial structure and configuration for a microfrontend application. This process typically includes creating the necessary directories, installing dependencies, and configuring the build tools to ensure that the microfrontend can run independently and integrate with other microfrontends or a shell application.


Would it be suitable in your opinion?

Reply:
I think its a perfect addition to the doc :)